### PR TITLE
New recipe: IgDiscover

### PR DESCRIPTION
**Pull request by marcelm**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

@bioconda/core: This is a new recipe; I would be glad for a review.

### CODE REVIEW

File: recipes/igdiscover/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+package:
+  name: igdiscover
+  version: "0.3"
+source:
+  fn: igdiscover-0.3.tar.gz
+  url: https://pypi.python.org/packages/ba/ff/bc398601c92d05bf502bcd3f49897b20cd3a0fe41cde0f17c8f073222ad4/igdiscover-0.3.tar.gz#md5=132e771db3e168018a613c0900b560d9
+  md5: 132e771db3e168018a613c0900b560d9
+requirements:
+  run:
+    - python >=3.3
```
Comment (johanneskoester) [inline recipes/igdiscover/meta.yaml]: Constraining the Python version here usually results in conda-build not pinning the Python version in the package. This is clearly a shortcoming in conda, but until it is solved, we have to omit the constraint here and rely solely on the skip option below.

File: recipes/igdiscover/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - scipy ==0.16.1
+    - pyyaml
+    - muscle
+    - pear
+    - flash
+    - igblast ==1.4.0
+    - vsearch ==1.9.7
+    - fastqc
+  build:
+    - python >=3.3
```
Comment (johanneskoester) [inline recipes/igdiscover/meta.yaml]: Same as above.

**Comment by johanneskoester** [comment]
Nice, is this the tool you have told me about at the ISMB, right? Feel free to merge after my comments have been addressed.

**Comment by marcelm** [comment]
Yes, that’s the tool! Thanks for the comments; I’ll address them.

**Pull request merged**